lib/screens/stops/stop.ex

```diff
@@ -339,4 +354,13 @@ defmodule Screens.Stops.Stop do
     |> Enum.uniq()
     |> Enum.into(%{})
   end
+
+  def get_all_routes_stop_sequence() do
+    @route_stop_sequences
+  end
+
+  def get_all_subway_stops() do
```

Comment:
Are duplicates something that should be taken out of this list? For example, each GL trunk stop is present in at least one branch, so those will appear more than once in the result of this function.

Reply:
Oh lol, I added this function but then ultimately didn't use it. Gotta get rid of it!